setup.py

```diff
@@ -12,6 +12,7 @@
     long_description_content_type="text/markdown",
     url="https://www.github.com/epispot/epispot/tree/nightly",
     packages=setuptools.find_packages(),
+    scripts=['bin/epispot'],
```

Comment:
You could also use `console_scripts` which will allow for more customizable command names.

```suggestion
    console_scripts=[
         'epi: bin/epispot'
    ]
```

Reply:
You should check though that this works with classes.

Reply:
It seems this doesn't work with classes. Ignore this suggestion.